README.md

```diff
@@ -2,10 +2,21 @@
 
 A module for Terasology.
```

Comment:
Actually, elaborate on this line a little more. Instead of just `A module for Terasology` change this to `A module for Terasology that causes a Gooey to randomly spawn around the world, enabling you to create Structures`. This makes the line a lot more meaningful.